annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/method/MethodDeclarationTree.java

```diff
@@ -45,8 +47,13 @@ public class MethodDeclarationTree extends MetaData<MethodNode> {
   /** Each method has a unique id across all methods. This hashmap, maps ids to nodes. */
   private HashMap<Integer, MethodNode> nodes;
 
+  /**
+   * A map from class flat name to a set of field names that are declared in that class but not
+   * initialized at declaration.
```

Comment:
Isn't this mapping class names to the constructors in the class?

Reply:
Sorry my mistake 70ec92849a5dd230d9fe4fc50ed604448d41ca83